public/app/core/components/Select/FolderPicker.tsx

```diff
@@ -6,13 +6,26 @@ import { useAsync } from 'react-use';
 
 import { AppEvents, SelectableValue, GrafanaTheme2 } from '@grafana/data';
 import { selectors } from '@grafana/e2e-selectors';
-import { useStyles2, ActionMeta, AsyncSelect, Input, InputActionMeta } from '@grafana/ui';
+import { useStyles2, ActionMeta, AsyncSelect, Input, InputActionMeta, Icon, Stack, Tooltip } from '@grafana/ui';
 import appEvents from 'app/core/app_events';
 import { contextSrv } from 'app/core/services/context_srv';
 import { createFolder, getFolderById, searchFolders } from 'app/features/manage-dashboards/state/actions';
 import { DashboardSearchHit } from 'app/features/search/types';
 import { AccessControlAction, PermissionLevelString } from 'app/types';
 
+const SlashesWarning = () => {
+  const styles = useStyles2(getStyles);
+  const onClick = () => window.open('https://github.com/grafana/grafana/issues/42947', '_blank');
```

Comment:
I don't think this is a good idea, it's ultimately something we should fix but sending users to a specific GitHub ticket seems weird as not every user of Grafana is technical enough to understand the context of the issue properly.

Reply:
That was an idea that came out from the UX feedback session. Everyone there agreed that this would be great in case the user wants to understand technically why this is happening @gillesdemey